webcompat/static/css/development/components/report-form.css

```diff
@@ -15,6 +15,19 @@
     text-align: center;
     justify-content: center;
     flex-wrap: wrap;
+    margin-top: 1em;
+  }
+
+  .wc-ReportForm .wc-UploadForm {
+    border: dashed 1px rgb(204, 204, 204);
+    border-radius: 20px;
+  }
+
+  .wc-ReportForm .r-Grid-cell {
+    margin-bottom: 1em;
+  }
+  .wc-ReportForm div.r-minM--1of2:nth-child(2) .wc-Form-group {
```

Comment:
when I was starting to touch to the form, I had the feeling to move everything to flexbox. with a very minimal markup. but it seems the `r-Grid-cell` is a css framework used for the forms? or site? Not sure. I didn't want to break everything given that I'm just occasionally editing it.